### PR TITLE
Applications

### PR DESCRIPTION
**Pull request by Crabcyborg**
Fixes https://github.com/Strategy11/formidable-pro/issues/3444

### CODE REVIEW

**Comment by garretlaxton** [comment]
For smaller screens, the **Product Review and Purchase** template title interferes with the **Upgrade Now** button.
Screenshot 2022-05-06 at 01-23-46 Formidable Applications ‹ 0xB9 Sec — WordPress

There are also no thumbnail images for the **Product Review and Purchase** and **Real Estate Listing** templates. Other than that, this pull looks good.

**Comment by Crabcyborg** [comment]
Thanks for bring that up @garretlaxton looks like there are a couple of new applications.

I talked to Steph about this issue and we went with a bit of a design change so it uses a button now on the line below. I added some text overflow to the title as well so it sticks to it's card.

Screen Shot 2022-05-06 at 1 05 55 PM

I'll add some thumbnails for these shortly as well. New templates will always be missing images though, as we're trying to bake those into the lite plugin (as WordPress has a rule about displaying images from our website).

**Review by garretlaxton — APPROVED** [review]

**Review by stephywells — COMMENTED** [review]
Looking great! I've got just a couple questions.

File: classes/controllers/FrmApplicationsController.php [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+class FrmApplicationsController {
+
+	/**
+	 * Add Applications menu item to sidebar and define Applications index page.
+	 *
+	 * @return void
+	 */
+	public static function menu() {
+		$label    = __( 'Applications', 'formidable' );
+		$new_pill = '<span class="frm-new-pill">NEW</span>';
```
[the remark this answers is not in the document]
Comment (Crabcyborg) [reply classes/controllers/FrmApplicationsController.php:18]: @stephywells we could, but I don't see an issue with just removing this in the future. The preview option doesn't have a date check either for the landing page option. I figure just keeping it in the release for a bit and then removing it manually keeps the code cleaner and less likely to just be forgotten.

File: classes/controllers/FrmApplicationsController.php [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+	}
+
+	/**
+	 * Sort applications alphabetically.
+	 *
+	 * @param array<array> $applications
+	 * @return array<array>
+	 */
+	private static function sort_templates( $applications ) {
+		usort(
```
[the remark this answers is not in the document]
Comment (Crabcyborg) [reply classes/controllers/FrmApplicationsController.php:98]: I'll log this for next release.

File: classes/models/FrmForm.php [hunk 3]
```diff
@@ -139,7 +139,7 @@ public static function after_duplicate( $form_id, $values ) {
 	/**
 	 * Switches field ID in fields.
 	 *
-	 * @since 5.2.08
+	 * @since 5.3
```
[the remark this answers is not in the document]
Comment (Crabcyborg) [reply classes/models/FrmForm.php:142]: That's just because Truong used another version and I changed it to the real version.

**Pull request merged**